src/main/java/at/hannibal2/skyhanni/data/IslandGraphs.kt

```diff
@@ -104,8 +102,9 @@ object IslandGraphs {
 
     val existsForThisIsland get() = currentIslandGraph != null
 
+    private var pathfindClosedNote: GraphNode? = null
     var closedNote: GraphNode? = null
-    var secondClosedNote: GraphNode? = null
+    private var secondClosedNote: GraphNode? = null
```

Comment:
i havent really been looking at too much graph stuff but to me these variable names seem wrong?
shouldnt these all be some form of "closestNode"? instead of closedNote?

Reply:
Agreed. will replace everything